torch/export/exported_program.py

```diff
@@ -424,6 +424,14 @@ def update_arg(old_arg, new_ph):
         ]
         output_specs = [
             OutputSpec(spec.kind, update_arg(spec.arg, new_outputs[i]), spec.target)
+            if spec.kind != OutputKind.USER_INPUT_MUTATION
+            else OutputSpec(
+                spec.kind,
+                update_arg(spec.arg, new_outputs[i]),
+                new_placeholders[
+                    i
```

Comment:
I don't think this is correct.. The target here is some placeholder node, not necessarily the ith placeholder node.

Reply:
Ah I thought it was that simple. I will leave the issue to others.